README.md

```diff
@@ -342,7 +342,7 @@ dependencies {
     implementation("pro.streem.pbandk:pbandk-runtime-jvm:0.10.0-SNAPSHOT")
 
     // For Android sourcesets/projects:
-    implementation("pro.streem.pbandk:pbandk-runtime-android:0.10.0-SNAPSHOT")
+    implementation("pro.streem.pbandk:pbandk-runtime:0.10.0-SNAPSHOT")
```

Comment:
Interesting, so now there is no android specific artifact anymore? 

We just publish all KMP artifacts and the `.module` files point Maven to the correct dependency. Is that correct?

It's a bit surprising that the other artifacts then still have specific names. 🤔

Reply:
Yeah, one of the improvements introduced by Kotlin 1.4 is that consumers of the library can now depend on the generic `pbandk-runtime` artifact and gradle's variant-aware dependency resolution figures out the correct dependency based on the `.module` file. This is actually required for the Android artifact in order to avoid having to specify different dependencies for each build type. The `pbandk-runtime-android` dependency is technically only for the `release` build type. In other words, Android consumers can either do:

```gradle
implementation("pro.streem.pbandk:pbandk-runtime:0.10.0-SNAPSHOT")
```

which will use gradle's variant-aware dependency resolution to choose between the `debug` and `release` versions of pbandk, or they can do:

```gradle
releaseImplementation("pro.streem.pbandk:pbandk-runtime-android:0.10.0-SNAPSHOT")
debugImplementation("pro.streem.pbandk:pbandk-runtime-android-debug:0.10.0-SNAPSHOT")
```

> It's a bit surprising that the other artifacts then still have specific names. 🤔

I think the other artifacts can likewise use the generic `pbandk-runtime` dependency. But I wanted to test that before updating the docs.

Reply:
Thanks for elaborating on that!